### DISCUSSION TITLE
VisualFBEditor 1.3.2

### DISCUSSION THREAD

**Discussion by XusinboyBekchanov** [XusinboyBekchanov/VisualFBEditor#276] [General]
Version 1.3.2 (April 16, 2022)
- Fixed: Click Set As Main menu item from Tab Window
- Fixed: Showing Functions List and Form/Code view toolbar on Windows XP and Classic without manifest mode
- Added: Bass examples
- Added: Manifest And Run As Administrator options to Project Properties
- Added: Add, Change, Remove and Clear buttons to New Project Dialog's Recent files tab
- Fixed: Run with cmd.exe and PowerShell.exe
- Fixed: Crash 64-bit IDE on open Chart Example
- Fixed: open the execute file like ld.exe in code windows when double click in output windows
- Fixed: Not showing the right selection text after replace in one line
- Added: DPI Awareness PerMonitorV2
- Fixed: The caption of Main Window not updated after save as a file
- Fixed: Edit control High DPI
- Added: Double Click on Edit control divider splits or joins window
- Added: Split vertically to Edit control
- Improved: Edit control proportionally changes the Divided position on control size changed
- Added: Window menu
- Added: Project preprocessor numbering
- Fixed: If the edit control separator is moved down from the edit control then the separator is moved back to the top.
- Improved: The Code editor can be split into two areas.
- Improved: Auto adds Designer property to the Form Constructor on Design
- Improved: Double-clicking a form in Explorer opens the Visual Designer.
- Fixed: The Index of Control array
- Added: Save and Open files with UTF8 (without BOM)
- Fixed: Tip of the Day
- Fixed: Edit Control Caret height on change Font Size with Ctrl+MouseWheel
- Fixed: Crash when changing a Form/Code view in a module without forms.

<hr /><em>This discussion was created from the release <a href='https://github.com/XusinboyBekchanov/VisualFBEditor/releases/tag/1.3.2'>VisualFBEditor 1.3.2</a>.</em>

**Comment by XusinboyBekchanov** [comment]
![image](https://user-images.githubusercontent.com/32607344/163696039-881d8671-0704-4e37-b4ea-72f9dc87fda0.png)

**Comment by flaviofornazier** [comment]
Hi all!


Congratulations to the author, co-workers and users of VisualFBEditor.

Could someone please send me a small "Windows Application" project
with only two forms? (e.g. a mainform calling another form).

I've tried everything, but I always receive the follow error after call
the second form using a command button:

Error: Variable not declared, Form2

I'm sure that I'm missing something...


Thank you in advanced.


_PS: Mr.Xusinboy Bekchanov, if you intend to add Portuguese language,
please just point me to the right direction and I'll be glad to do it.
There is no a substantial difference between 5th and 9th most commonly
spoken languages. They revolve around 250 millions of speakers._

**Reply by hustbeef** [reply]
Yes. There are two forms modes, one is MDI forms and the another is single forms. I'd like to post a exzample files on freeBasic forums . https://www.freebasic.net/forum/viewtopic.php?p=293486&sid=a25c91fb4b56ec926742927f9f15f9b1#p293486

**Reply by XusinboyBekchanov** [reply]
I answered this question on the forum:
https://freebasic.net/forum/viewtopic.php?p=293501#p293501

**Comment by XusinboyBekchanov** [comment]
![image](https://user-images.githubusercontent.com/32607344/185759804-8b8522c6-49c7-43c3-85d7-668a4bcae84d.png)

**Reply by hustbeef** [reply]
Now。 "notset" is useless annd maybe can be replace by "EXE" or "DLL" buttom
